include/deal.II/lac/scalapack.templates.h

```diff
@@ -691,13 +691,13 @@ extern "C"
  * wrappers for other numeric types can be added in the future
  */
 template <typename number>
-inline void Cgerv2d(int context,
-                    int M,
-                    int N,
-                    number *A,
-                    int lda,
-                    int rsrc,
-                    int csrc)
+inline void Cgerv2d(int /*context*/,
+                    int /*M*/,
+                    int /*N*/,
+                    number */*A*/,
```

Comment:
Would you mind to separate the pointer sepecifier `*` from the comment by a whitespace here and below?

Reply:
sure.